css/css-flexbox/gap-001-lr-ref.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Flexbox Test: gap - horizontal with LR vertical writing mode</title>
+<link rel="author" title="Adam Argyle" href="mailto:argyle@google.com">
+<style>
+  body {
+    writing-mode: vertical-lr;
+  }
+
+  section {
+    background-color: red;
```

Comment:
I'd make this a different color than red, as red == bad.

Reply:
the rendered test is mostly green lol, where the red is the gap instead of the red being the big blocks. aka, the visual result is dominated by green, but yeah, red is used for the gap. thoughts here? i'm consistent in my usage of this strategy, but am willing to adjust and align my tests to the standards of the repo.

Reply:
I agree with @bfgeek using red as an expected output is somehow confusing. You could use any other color for the gaps like blue, yellow, grey, whatever... (I don't have a great suggestion). But red is kind of confusing.

Reply:
biggest bummer of this is, i'd switched them (red and green usage) over halfway through authoring from some other similar feedback! consistency is more important than the color of the gap though yeah?  

the [assertion text specifies what passing and not passing are](https://github.com/web-platform-tests/wpt/pull/20414/files/0e6b1388f55b061f9cc579ad573119305e9d6ad6#diff-b8d74e5eb98041a53e20365d6cf3d50cR25) as well, in case there are questions about what's being tested..

Reply:
It's pretty unequivocal in this repo that red == bad. Can you just 
`sed -ie 's/red/lightblue/g' *html`, or something?

Reply:
Yes, "red means a failing test" is a *hard* rule; I will absolutely reject any tests which try to show red while passing.

Reply:
alright, just so i dont have to adjust colors on all these tests again... please confirm the following change to colors:
- green: where we expect gap to be
- red: do i use red anywhere?
- boxes: use lightblue?

Reply:
1. Use red if and only if its appearance indicates a failure.
2. Ideally, use only green otherwise.
3. Other colors are fine if necessary, but the more you have to read to understand success vs failure the worse it is. If you can make it "any green means success", that's great.

So just swapping your test to use green for the gaps and a non-red for the boxes is fine, with an explanation that lack of green is a failure. (Gray or blue is pretty common as a "neutral" color that indicates an unimportant part of the test.)

An *ideal* test would show red on failure, like making the flex items red and then absposing some green boxes exactly over them, so if the property isn't supported the red will show inbetween them. An even more ideal test would then make the whole test match one of the standard reftest refs, like the 100x100 green square. ^_^

(That test would pass if the gap was equal *or greater* than expected; it'll only fail if the gap is less than expected or 0. You can combine it with a test that the gap isn't larger than expected by swapping colors - make the gap red and the items green, then abspos a green box over the gap.)

(Example test exercising both of those: <https://codepen.io/TabAtkins/pen/bGNyEKw>)

Reply:
worked with tab via chat, updated all tests to colors that are more consistent with all the others 👍